packages/arb-token-bridge-ui/src/components/TransferPanel/TokenImportDialog.tsx

```diff
@@ -18,6 +18,8 @@ import { useERC20L1Address } from '../../hooks/useERC20L1Address'
 import { Dialog, UseDialogProps } from '../common/Dialog'
 import { SafeImage } from '../common/SafeImage'
 import { getExplorerUrl } from '../../util/networks'
+import Image from 'next/image'
+import GrumpyCat from '@/images/grumpy-cat.webp'
```

Comment:
can we unify the import root?
some are @ and some are ../

Reply:
Some were not in the image folder, so I didn't catch them, will move them and update